docs/auth.md

```diff
@@ -190,8 +190,8 @@ the following configuration options to your `onFlow()`:
 ```ts
 import { onFlow } from '@genkit-ai/firebase/functions';
 
-export const jokeFlow = onFlow({
-    name: 'jokeFlow',
+export const menuSuggestionFlow = onFlow({
+    name: 'menuSuggestionFlow',
```

Comment:
Reference to `(joke)` a few more lines below as an input param under authPolicy.